src/moveFile.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const fs = require('fs').promises;
+const path = require('path');
+
+async function moveFile(fileName, moveTo) {
+  const filePath = path.join(__dirname, fileName);
+  let movePath = path.join(__dirname, '..', moveTo);
+  let data;
+
+  movePath = movePath.endsWith('/') ? movePath.slice(0, -1) : movePath;
+
+  try {
+    fs.readFile(movePath, 'utf-8');
```

Comment:
fs.readFile is an async function
```suggestion
    await fs.readFile(movePath, 'utf-8');
```